frontend/src/App.tsx

```diff
@@ -1,17 +1,16 @@
-import React from "react";
-import LandingPage from "./components/landing";
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import LandingPage from './components/landing';
+import Table from "./components/table";
 
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import "./App.css";
+import './App.css';
 
-const App = () => {
-  return (
-    <Router>
-      <Switch>
-        <Route exact path="/" component={LandingPage} />
-      </Switch>
-    </Router>
-  );
-};
+const App = () => (
+  <Router>
+    <Switch>
+      <Route exact path="/" component={Table} />
```

Comment:
Once I tell you that this PR is ready to be merged with master, change this line to lead to the LandingPage.